services/vpc_manager/src/main/java/com/futurewei/alcor/vpcmanager/controller/DebugVpcController.java

```diff
@@ -92,25 +91,26 @@ public Map getVpcCount() throws CacheException {
             method = POST,
             value = {"/debug/project/{projectid}/vpcs"})
     @ResponseStatus(HttpStatus.CREATED)
-    public VpcStateJson createVpcState(@PathVariable String projectid, @RequestBody VpcStateJson resource) throws Exception {
-        VpcState vpcState = null;
+    public VpcWebJson createVpcState(@PathVariable String projectid, @RequestBody VpcWebRequestJson resource) throws Exception {
+        VpcWebResponseObject inVpcState = new VpcWebResponseObject();
 
         try {
             RestPreconditionsUtil.verifyParameterNotNullorEmpty(projectid);
 
-            VpcState inVpcState = resource.getVpc();
+            VpcWebRequestObject vpcWebRequestObject = resource.getNetwork();
+            BeanUtils.copyProperties(vpcWebRequestObject, inVpcState);
             RestPreconditionsUtil.verifyResourceNotNull(inVpcState);
             RestPreconditionsUtil.populateResourceProjectId(inVpcState, projectid);
 
             Transaction transaction = this.vpcDatabaseService.getCache().getTransaction();
```

Comment:
Do we have to use a GetTransaction for every get request? If so, please move the transaction block code to service layer, instead of keeping it in Controller layer.

Reply:
Okay ~ I'll revise it in next update